framework/SupervisedLearning/pickledROM.py

```diff
@@ -18,39 +18,36 @@
   Originally from SupervisedLearning.py, split in PR #650 in July 2018
   Specific ROM implementation for pickledROM
 """
-#for future compatibility with Python 3--------------------------------------------------------------
-from __future__ import division, print_function, unicode_literals, absolute_import
-#End compatibility block for Python 3----------------------------------------------------------------
-
-
-#External Modules------------------------------------------------------------------------------------
-#External Modules End--------------------------------------------------------------------------------
-
-#Internal Modules------------------------------------------------------------------------------------
 from .SupervisedLearning import supervisedLearning
-#Internal Modules End--------------------------------------------------------------------------------
-
 
 
 class pickledROM(supervisedLearning):
   """
     Placeholder for ROMs that will be generated by unpickling from file.
   """
-  def __init__(self, messageHandler=None, **kwargs):
+  def __init__(self, **kwargs):
     """
       A constructor that will appropriately intialize a supervised learning object
-      @ In, messageHandler, MessageHandler object, optional, it is in charge of raising errors, and printing messages
       @ In, kwargs, dict, an arbitrary list of kwargs
       @ Out, None
     """
-    self.printTag          = 'pickledROM'
-    self.messageHandler    = messageHandler
-    self._dynamicHandling  = False
-    self.initOptionDict    = {}
-    self.features          = ['PlaceHolder']
-    self.target            = 'PlaceHolder'
-    self.metadataKeys = set() # keys that can be passed to DataObject as meta information
-    self.metadataParams = {}  # indexMap for metadataKeys to pass to a DataObject as meta dimensionality
+    super().__init__(**kwargs)
+    self.printTag = 'pickledROM'
+    self._dynamicHandling = False
+    self.initOptionDict = {}
+    self.features = ['PlaceHolder']
+    self.target = 'PlaceHolder'
+    self.metadataKeys = set()
+    self.metadataParams = {}
```

Comment:
why comments are removed for these self variables?

Reply:
they were actually meant to be removed entirely, since the base class defines them. Removed.